### ISSUE TITLE
Work with TTS Tech Portfolio to obtain AWS DMS permissions needs

### ISSUE THREAD

**Issue by carjug** [HHS/OPRE-OPS#72]
As engineers, we would like to be able to test our DB migration strategy in the TTS sandbox if at all possible. The default permissions do not allow us to use AWS DMS in the way we would like to. In order to get these permissions amended, we have to talk to the TTS Tech Portfolio.

**Tasks**
- [x] Set up meeting with TTS Tech Portfolio
- [x] Record results of meeting here
- [ ] Verify that "alternative" accounts they set  us up with have the appropriate permissions (can create replication  env)

**Comment by amymok** [comment]
@carjug, this is a result from #61, do you want to continue with asking TTS Tech Portfolio?  Or do you prefer I continue with this one, since this is a DB related task.

**Comment by carjug** [comment]
Oh I figured I'd take this one! And I can meet with you to share results.

**Comment by carjug** [comment]
Ongoing conversation in Slack about this... looks like we might get our permissions changed to do what we need to do after all. See this thread for details:
https://gsa-tts.slack.com/archives/C039MHHF8/p1626127776006200

**Comment by carjug** [comment]
Not sure what to do with this one...still waiting on the "alternative" accounts for Amy and I that were proposed in the above-linked Slack thread, but I still have permissions to do the things I need to in AWS. Basically, at this point I'd say this ticket is done given the tasks, but that we're waiting on them for final word. I'll follow up with them again on Monday.

**Comment by ninamak** [comment]
@amymok Can you check that you have the appropriate permissions using your current account (you can set up a replication environment)?

**Comment by amymok** [comment]
@carjug I checked that I was able to "Create replication instance" under DMS.  I didn't actually press the button to create the instance, but the button is not grayed out.  Anything else I should check, or is this sufficient?

**Comment by carjug** [comment]
@amymok click "Create", it is at that point that you will find out whether you have a proper IAM role.

**Comment by amymok** [comment]
@carjug I clicked on it, and it is starting to make the instance, at least it is saying <replication name> creation in progress.

**Comment by carjug** [comment]
Ok @amymok it looks like you have the right permissions. If you didn't it would show something like this
<img width="841" alt="Screen Shot 2021-07-13 at 9 15 04 AM" src="https://user-images.githubusercontent.com/10871517/127359213-c93d7afd-68ec-4611-9106-22b5aae6ac7f.png">

**Comment by ninamak** [comment]
Alternative accounts still have not been set up, but we have the permissions we need with our existing accounts.  So we're closing this out.  If things change, we can open a new issue.
